app/models/smart_proxy.rb

```diff
@@ -172,6 +172,6 @@ def get_features
   end
 
   class Jail < ::Safemode::Jail
-    allow :id, :name
+    allow :id, :name, :hostname, :hosts_count
```

Comment:
Can it be a problem that Katello also defines the hostname?
https://github.com/Katello/katello/blob/a9abccc2c10f5f2c1194d564834d131d57b28401/app/models/katello/concerns/smart_proxy_extensions.rb#L336-L338

Either way, after merging this that can be removed.

Reply:
why do we need the hosts_count?

Reply:
I will do a followup PR into Katello.

That `hosts_count` was added by accident, not needed.

Reply:
Done: https://github.com/Katello/katello/pull/8748